src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -1458,11 +1470,12 @@ private void ShowInvocationExamples()
 
             List<ITemplateInfo> templateList = _matchedTemplates.Select(x => x.Info).ToList();
             Reporter.Output.WriteLine("Examples:");
+            HashSet<string> usedGroupIds = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
 
             foreach (string preferredName in preferredNameList)
             {
                 ITemplateInfo template = templateList.FirstOrDefault(x => string.Equals(x.ShortName, preferredName, StringComparison.OrdinalIgnoreCase));
-                if (template != null)
+                if (template != null && usedGroupIds.Add(template.GroupIdentity ?? string.Empty))
```

Comment:
This will cause all templates with no group identity to be eliminated from possible use after the first one is used.